.github/workflows/l10n-sync.yml

```diff
@@ -0,0 +1,28 @@
+name: Fetch latest strings from the l10n repo
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
can we add a comment saying what this schedule is? I.e., `every night at midnight UTC` ?

Reply:
Good point: d76ac61